ecosystem/sep-0043.md

```diff
@@ -0,0 +1,146 @@
+## Preamble
+
+```
+SEP: 0043
+Title: Standard Web Wallet API Interface
+Authors: Piyal Basu <piyal@stellar.org>, Leigh McCulloch <@leighmcculloch>, George Kudrayvtsev <george@stellar.org>, Enrique Arrieta <@earrietadev>, Orbit Lens <@orbitlens>
+Track: Standard
+Status: Draft
+Created: 2024-04-11
+Version: 1.2.0
+Discussion: https://github.com/stellar/stellar-protocol/discussions/1467
+```
+
+## Abstract
+
+This standard aims to enhance developer experiences by standardizing an interface for web wallet's to use when
+interacting with other codebases.
+
+## Motivation
+
+There are numerous Stellar wallets in the ecosystem with this number expected to grow as the ecosystem does. Wallets
+expose an API to allow dapps to call methods that preform such common tasks as transmitting a wallet's address. An issue
+is that wallets all use slightly different API's to preform similar functions, with different method names and call
+signature as well as different return values. Developers have a motivation to integrate with as many as possible, but
```

Comment:
```suggestion
signatures as well as different return values. Developers have a motivation to integrate with as many as possible, but
```

Reply:
Thanks - the lack of spell check in vscode is very evident in this PR 😬